llif/network.go

```diff
@@ -0,0 +1,20 @@
+package llif
```

Comment:
is the if in llif for interface? if yes, then don't really understand why is the fs implementation inside of the llif/ dir.

Reply:
That's true. Let me put them in a separate `llmodules` instead. Does that sound better?

Reply:
Done!